packages/vue3-transpiler/src/templateLoader.ts

```diff
@@ -33,6 +34,7 @@ function TemplateLoader(source: string, loaderContext: LoaderContext) {
   }
 
   const compiled = compileTemplate({
+    id: uuid(),
```

Comment:
What is this id for?

Reply:
A new dependency from `@vue/compiler-sfc`

Reply:
Doesn't seem like we're using vue/compiler-sfc, this file has an import for https://unpkg.com/browse/vue3-browser-compiler@1.0.0/ so probably didn't change there as that module has never been updated?

Also still dunno what the id is for, if it is used for hmr we can't use uuid's that's what I was trying to get to, than it should be the module id, which is the filepath with some optional query